packages/cli/src/api/pseudoLocalize.test.ts

```diff
@@ -92,4 +92,14 @@ describe("PseudoLocalization", () => {
     expect(pseudoLocalize("replace {count}")).toEqual("ŕēƥĺàćē {count}")
     expect(pseudoLocalize("replace { count }")).toEqual("ŕēƥĺàćē { count }")
   })
+
+  it("multiple plurals is wrong45", () => {
+    expect(
+      pseudoLocalize(
+        "{bcount, plural, one {boy} other {# boys}} {gcount, plural, one {girl} other {# girls}}"
+      )
+    ).not.toEqual(
+      "{bcount, plural, one {ƀōŷ} other {# ƀōŷś}} {gcount, plural, one {ĝĩŕĺ} other {# ĝĩŕĺś}}"
+    )
+  })
 })
```

Comment:
Consider adding a test for `compile` with `shouldPseudolocalize = true`, with ICU strings, and without ICU strings.